### PR TITLE
Create DCA632FFFE7697E0.json

### PR DESCRIPTION
**Pull request by panobs**

### CODE REVIEW

File: dca632fffe7697e0.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
+{
+  "gateway_conf": {
+    "gateway_ID": "dca632fffe7697e0",
```
Comment (gonzalocasas) [inline dca632fffe7697e0.json]: This EUI and also the file name (not extension) should be all uppercase, please update and then we can merge.

File: dca632fffe7697e0.json [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+    "servers": [
+      {
+        "server_address": "router.eu.thethings.network",
```
Comment (gonzalocasas) [inline dca632fffe7697e0.json]: BTW, if you're in Switzerland, you might want to consider using the swiss backend: `ttn.opennetworkinfrastructure.org` (https://www.thethingsnetwork.org/docs/gateways/packet-forwarder/semtech-udp.html#router-addresses).

File: dca632fffe7697e0.json (hunk 1 above)
Comment (panobs) [reply dca632fffe7697e0.json]: OK

**Comment by gonzalocasas** [comment]
Thanks!! Merged!

**Pull request merged**